### PR TITLE
Add ability to launch Redux DevTools Chrome extension

### PR DESCRIPTION
**Pull request by tsareg**
This adds ability to use Redux DevTools Chrome extension, which might be very helpful for development purposes. You can read more about it at https://github.com/zalmoxisus/redux-devtools-extension.

### CODE REVIEW

File: index.web.js [hunk 1]
```diff
@@ -48,10 +48,12 @@ const reducer = combineReducers({
...
-const store = createStore(reducer, applyMiddleware(
-    Thunk,
-    router,
-    routerMiddleware(browserHistory)
+const store = createStore(reducer, compose(
+    applyMiddleware(
+        Thunk,
+        router,
+        routerMiddleware(browserHistory)),
+    window.devToolsExtension ? window.devToolsExtension() : f => f
```
Comment (lyubomir) [inline index.web.js:53]: I don't understand how people read such an indentation of function arguments.

**Comment by lyubomir** [comment]
Could we please have a description on this PR? I don't understand what it does (in addition to my previous comment on the indentation of the source code).

**Comment by lyubomir** [comment]
Why is this specific to the Web version of the application? Cannot we have it on React Native?

**Comment by tsareg** [comment]
@lyubomir it's Chrome extension that's why it's only for web. Again, this is a development thing, we can hide it under **DEV** flag or something like that.

**Comment by tsareg** [comment]
I don't think this is relevant anymore. It seems no one else wants it and personally I can always use this locally if I need it.

**Comment by lyubomir** [comment]
I don't understand why it's not relevant anymore. I do want a development tool to aid me in debugging the Redux state. Especially, after one of the arguments for adopting Redux in the first place was that it had good development environment support. @tsareg, could you please prepare for a brief presentation of the devtools of your choice over a desktop stream so that we can learn how to use that tool?

**Comment by lyubomir** [comment]
@tsareg, I kind of merged this PR by pushing into master a piece of source code which I believe to be functionally equivalent to yours. Please make sure that it achieves your goal. I'd still like you to give us a brief presentation on the subject of its use though.

**Pull request closed without merging**